packages/server/src/util.js

```diff
@@ -1,5 +1,6 @@
 export const clearModuleCache = moduleName => {
-  const m = require.cache[moduleName]
+  const cache = (typeof __non_webpack_require__ !== 'undefined' ? __non_webpack_require__ : require).cache;
```

Comment:
Sorry for being picky, but let's safe future developers time and make it right this time

Let's use "single source" for both operations if the HAVE to be synchronized.
```js
const getRequire = () => typeof __non_webpack_require__ !== 'undefined' ? __non_webpack_require__ : require;

export const clearModuleCache = moduleName => {
  const cache = getRequire().cache; // 👈
...
}

export const smartRequire = modulePath => {
  if (process.env.NODE_ENV !== 'production' && module.hot) {
    clearModuleCache(modulePath)
  }
 
  return getRequire(moduleName);  // 👈
}
```